projects/quickjs/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://bellard.org/quickjs/"
+primary_contact: "fabrice@bellard.org"
+auto_ccs :
+- "p.antoine@catenacyber.fr"
+
+sanitizers:
+- address
```

Comment:
You need to add i386 to architectures to get i386 fuzzing btw.

Reply:
Indeed, I wanted to get the project running first, and then expand it...